tools/check_output_test.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# Usage: check_output_test.py [path to deno executable]
+import os
+import sys
+import subprocess
+
+root_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+tests_path = os.path.join(root_path, "tests")
+
+
+def check_output_test(deno_fn):
```

Comment:
`fn` means function to me. After studying the code, I presume it means "filename" in this context.
Rename to `deno_runtime` or `deno_exe` or `deno_bin`.